function sample() {
  try {
    console.log("try catch");
  } catch (error) {
    console.log("error Block");
  }
}
sample();
